Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenButton.cs

```diff
@@ -11,6 +11,7 @@ namespace UnityEngine.InputSystem.OnScreen
     /// input.
     /// </summary>
     [AddComponentMenu("Input/On-Screen Button")]
+    [HelpURL("https://docs.unity3d.com/Packages/com.unity.inputsystem@latest?subfolder=/manual/OnScreen.html%23on-screen-buttons")]
```

Comment:
Should we use @latest or can we somehow link it to the actual package version that is in the project? This may take them to a page that is incorrect for the version they are using

Reply:
Maybe do something like this: Define the version in C# somewhere as a const:
class Version
{
    public const string kVersion = "1.0";
}

...
[HelpURL("https://docs.unity3d.com/Packages/com.unity.inputsystem@" + Version.kVersion + "?subfolder=/manual/OnScreen.html%23on-screen-buttons")]




[HelpURL("some url" + Version.kVersion)]

Reply:
https://gist.github.com/karljj1/0c9b8fa8d737dc69f810f6ec90ea83ab

Reply:
Good point. Will change. Also, occurs to me I should probably move the entire first part of that string all the way up to the actual doc path into a string const. I'll put it in AssemblyInfo.cs so that the stuff including the version is all in one place.

Reply:
Ok, went one step further and also locked this in with tests.